### PR TITLE
fix: update eventarc storage name

### PR DESCRIPTION
**Pull request by grant**
Uses "Cloud Storage" instead of "GCS" for Eventarc Audit quickstart per branding guidelines.

See internal b/180109929

### CODE REVIEW

**Comment by grant** [comment]
(+ Averi who reviewed similar PRs)

**Review by averikitsch — APPROVED** [review]

File: eventarc/audit_storage/main.go [hunk 1]
```diff
@@ -24,9 +24,9 @@ import (
 	"os"
 )
 
-// HelloEventsStorage receives and processes a Pub/Sub message via a CloudEvent.
+// HelloEventsStorage receives and processes a Cloud Audit Log events with Cloud Storage data.
```
Comment (tbpg) [inline eventarc/audit_storage/main.go:27]:
```suggestion
// HelloEventsStorage receives and processes a Cloud Audit Log event with Cloud Storage data.
```

**Review by tbpg — APPROVED** [review]
One nit, otherwise LGTM.

File: eventarc/audit_storage/main.go (hunk 1 above)
Comment (grant) [reply eventarc/audit_storage/main.go]: Done.

**Pull request merged**